server/src/server.ts

```diff
@@ -3,18 +3,25 @@ import type { TwilioCredentials } from './common/types';
 import { createTokenRoute } from './routes/token';
 import { createTwimlRoute } from './routes/twiml';
 import { createLogMiddleware } from './middlewares/log';
+import { auth } from 'express-oauth2-jwt-bearer';
+import config from '../config';
 
 export function createExpressApp(twilioCredentials: TwilioCredentials) {
   const app = express();
 
+  const jwtCheck = auth({
+    audience: config.audience,
+    issuerBaseURL: config.issuerBaseURL,
+  });
+
   app.use(express.urlencoded({ extended: false }));
   app.use(express.json());
 
   app.use(createLogMiddleware());
 
   const tokenRouter = Router();
   tokenRouter.use(createTokenRoute(twilioCredentials));
-  app.post('/token', tokenRouter);
+  app.post('/token', jwtCheck, tokenRouter);
```

Comment:
Oh this is cool! This means any endpoint that we want to protect, we just do this same thing?

Reply:
Yup we would just need to add `jwtcheck` to each route we want to protect

Reply:
Not for this PR, but is it possible to get other user information such as email or username from the jwt? Or can you post an example jwt?

Reply:
Ok, we can take a look at that later. But can you please post an example jwt here? I like to see it's structure.